ftw/activity/browser/templates/activity_raw.pt

```diff
@@ -2,6 +2,7 @@
 
   <div class="activity">
     <div class="events"
+         tal:attributes="data-total python:len(view._lookup())"
```

Comment:
I did not yet have the time to look into that.
The problem is also not mission critical, therefore I'm not prioritizing this issue.
I have it on my radar though 😉